packages/asset-swapper/src/utils/market_operation_utils/sampler_operations.ts

```diff
@@ -35,6 +36,19 @@ import {
     UniswapV2FillData,
 } from './types';
 
+/**
+ * Source filters for `getTwoHopBuyQuotes()` and `getTwoHopSellQuotes()`.
+ */
+export const TWO_HOP_SOURCE_FILTERS = SourceFilters.all().exclude([
+    ERC20BridgeSource.MultiHop,
+    ERC20BridgeSource.MultiBridge,
+    ERC20BridgeSource.Native,
+]);
+/**
+ * Source filters for `getSellQuotes()` and `getBuyQuotes()`.
+ */
+export const BATCH_SOURCE_FILTERS = SourceFilters.all().exclude([ERC20BridgeSource.MultiHop, ERC20BridgeSource.Native]);
```

Comment:
So now the operations will do some extra filtering of sources passed in so the consumer doesn't have to worry about it.